analyze.sh

```diff
@@ -8,14 +8,14 @@ ARTIFACT_DIR=${ARTIFACT_DIR:=/tmp/artifacts}
 
 cd frontend
 echo "Analyzing Webpack bundles..."
-yarn run analyze --no-open
```

Comment:
@spadgett FYI, with this PR, running `yarn analyze` won't open the generated HTML report in default browser.